reference/src/validity/unions.md

```diff
@@ -0,0 +1,13 @@
+# Validity of unions
+
+**Disclaimer**: This chapter is a work-in-progress. What's contained here
+represents the consensus from issue [#73]. The statements in here are not (yet)
+"guaranteed" not to change until an RFC ratifies them.
+
+## Validity of unions with zero-sized fields
+
+A union containing a zero-sized field can contain any value. An example of such
```

Comment:
```suggestion
A union containing a zero-sized field can contain any bit pattern. An example of such
```

Reply:
BTW: still not sure if that's the best way to put it but this change is at least an improvement